src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -153,13 +155,16 @@ function FloatingActionButtonAndPopover(props) {
 
     const quickActionReport = useMemo(() => (props.quickAction ? ReportUtils.getReport(props.quickAction.chatReportID) : 0), [props.quickAction]);
 
+    const policy = props.allPolicies ? props.allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${quickActionReport.policyID}`] : undefined;
```

Comment:
```suggestion
    const quickActionPolicy = props.allPolicies ? props.allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${quickActionReport.policyID}`] : undefined;
```
To keep things consistent :wink: